docs/README.md

```diff
@@ -683,6 +684,24 @@ jobs:
           state: success
 ```
 
+### github-deployments-delete
+
+Delete gitHub deployments.Used as workaround to delete the flood of messages added to PRs when we run a workflow with a matrix that have the field environment set in the job.
+In this case a new comment is added in PR for every entry of the matrix.
+
+Sample usage:
+
+```yaml
+   permissions:
+    deployments: write # This is required for deployment statuses management
+
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/github-deployments-delete@ref
+        with:
+          branch-name: ${{ github.head_ref }}
+
+```
```

Comment:
```suggestion
```yaml
  permissions:
    deployments: write # This is required for deployment statuses management

  steps:
    - uses: Alfresco/alfresco-build-tools/.github/actions/github-deployments-delete@ref
      with:
        branch-name: ${{ github.head_ref }}
```
```

Reply:
Sorry I'm not sure the suggestion feature picked up the right things in above comment: Iwanted to fix indent (permissions and steps at same level) + remove extra blank line at the end

Reply:
Corrected.